projects/storefrontlib/src/shared/components/replenishment-order-cancellation-dialog/replenishment-order-cancellation-dialog.component.spec.ts

```diff
@@ -99,6 +101,10 @@ describe('ReplenishmentOrderCancellationDialogComponent', () => {
 
   it('should redirect to same page and add global message on successful cancellation ', () => {
     spyOn(userReplenishmentOrderService, 'cancelReplenishmentOrder').and.stub();
+    spyOn(
+      userReplenishmentOrderService,
+      'clearCancelReplenishmentOrderProcessState'
+    ).and.callThrough();
```

Comment:
Not a huge thing, but you don't need the `callThrough` since you are just testing that it's called and not the actions taken.